### ISSUE TITLE
KeyError: 'expires_at' on OAuth2 callback for GitHub

### ISSUE THREAD

**Issue by MatthewScholefield** [frankie567/fastapi-users#431]
## Describe the bug

`KeyError: 'expires_at'` on OAuth2 callback for GitHub:

```
  File ".../lib/python3.8/site-packages/fastapi_users/router/oauth.py", line 111, in callback
    expires_at=token["expires_at"],
               └ {'access_token': '5891b5b...', 'token_type': 'bearer', 'scope': ''}

KeyError: 'expires_at'
```

## To Reproduce

 - Create the example OAuth2 project with GitHub instead of Google OAuth
 - Navigate to `/auth/github/callback?auth_backend=jwt` (or equivalent)
 - Follow the `authorization_url` and grant your application access to your GitHub account

## Expected behavior

`null` response and a new user is created through GitHub OAuth2.

## Configuration

 - Python version: 3.8.6
 - FastAPI version: 0.62.0
 - FastAPI Users version: 3.1.2

## Discussion

GitHub [doesn't provide an expiration](https://docs.github.com/en/free-pro-team@latest/developers/apps/authorizing-oauth-apps#2-users-are-redirected-back-to-your-site-by-github). Since it looks like the token expiration isn't ever explicitly used (seems to only be there to keep track of), it seems like the only change necessary to fix this is to make `expires_at` optional in `BaseOAuthAccount`. However, while I'm not sure, I'm assuming any schema change (even widening like this) would break tortoise and sqlalchemy. If this is the case, is the best solution just adding a default value of 0? (ie. `token.get('expires_at', 0)`)

**Comment by frankie567** [comment]
Thank you for the detailed report! That's indeed a case I've not foreseen. The fix should indeed be to make this property optional everywhere, even in databases schemas, which will require a migration.

I'll take care of this rapidly!

**Comment by frankie567** [comment]
Fixed in [`v4.0.0`](https://github.com/frankie567/fastapi-users/releases/tag/v4.0.0) 🎉

**Comment by frankie567** [comment]
@all-contributors add @MatthewScholefield for bug
